src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -121,11 +121,20 @@ public static WorkspaceManagerService fromContext() {
         Context.requireUser().getUserAccessToken(), Context.getServer());
   }
 
+  /**
+   * Factory method for class that talks to WSM. Pulls the current server and user from the context.
+   * Uses the pet SA's credentials.
```

Comment:
probably worth saying why a client would want to use this instead of `fromContext`

Reply:
added